sources/npmRegistryUtils.ts

```diff
@@ -26,7 +25,9 @@ export async function fetchAsJson(packageName: string) {
     headers.authorization = `Basic ${encodedCreds}`;
   }
 
-  return httpUtils.fetchAsJson(`${npmRegistryUrl}/${packageName}`, {headers});
+  const response = await httpUtils.fetch(`${npmRegistryUrl}/${packageName}`, {headers});
+  const data: any = await response.json();
+  return data;
```

Comment:
nit: It seems to me we could keep the `fetchAsJson` util, since we're hiding away e.g. the status code check behind `httpUtils.fetch`, we might as well go the extra mile and hide the `response.json()` call as well, and get a smaller diff.